### DISCUSSION TITLE
beta projects

### DISCUSSION THREAD

**Discussion by robphilipp** [community/community#9544] [Projects and Issues]
I really like the project sheets. One small comment: When selecting "Assignees", the only way to have it "stick" is to use the mouse or press "escape". The escape key is unintuitive. Can the "return" key finish the selection and give another line? This would make the flow much cleaner.

**Comment by MichaelTiemannOSC** [comment]
Suggest changing the title to more descriptive "Suggest more intuitive way to make assignments via Assignees".
